lib/logatron/configuration.rb

```diff
@@ -30,12 +30,7 @@ def initialize
       @transformer =  proc {|x| x.to_json}
       @host = `hostname`.chomp
       @level = INFO
-      levels = [DEBUG,
-                INFO,
-                WARN,
-                ERROR,
-                CRITICAL,
-                FATAL]
+      levels = Logatron::SEVERITY_MAP.keys
```

Comment:
Better, but this still unnecessarily assumes that the map elements are defined in the same order as their values. Filter based on `SEVERITY_MAP` values instead of the `Array#slice` hack below.